qa-tests-backend/src/test/groovy/PolicyConfigurationTest.groovy

```diff
@@ -14,6 +14,7 @@ import io.stackrox.proto.storage.ScopeOuterClass.Scope
 import common.Constants
 import groups.BAT
 import groups.SMOKE
+import groups.COMPATIBILITY
```

Comment:
this isn't used?

Reply:
Good catches, I had added tests from these two files to COMPATIBILITY then removed them again later.